2017-10-31 18:47:24 [DEBUG][AuthenticationService] [isUserAuthentification:49] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, Р°СѓС‚РµРЅС‚РёС„РёС†РёСЂРѕРІР°РЅ Р»Рё РїРѕР»СЊР·РѕРІР°С‚РµР»СЊ
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [isUserAuthorization:28] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, Р°РІС‚РѕСЂРёР·РѕРІР°РЅ Р»Рё РїРѕР»СЊР·РѕРІР°С‚РµР»СЊ
2017-10-31 18:47:24 [INFO ][DataContextDAO] [isUserRoleValid:106] - Р РѕР»СЊ РІРµСЂРЅР°
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [isUserAccounting:72] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, РІС‹РїРѕР»РЅРµРЅ Р»Рё РїСЂРѕС†РµСЃСЃ Р°РєРєР°СѓРЅС‚РёРЅРіР°
2017-10-31 18:47:24 [INFO ][DataContextDAO] [isDateDsAndDeValid:128] - Р”Р°С‚Р°(С‹) СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‚ С„РѕСЂРјР°С‚Сѓ
2017-10-31 18:47:24 [INFO ][DataContextDAO] [isVolumeValid:146] - Р—РЅР°С‡РµРЅРёРµ РѕР±СЉС‘РјР° РІРІРµРґРµРЅРЅРѕ РєРѕСЂСЂРµРєС‚РЅРѕ!
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [addAccounting:52] - Р”РѕР±Р°РІР»РµРЅРёРµ РїРѕР»СЊР·РѕРІР°С‚РµР»СЊСЃРєРѕРіРѕ СЃРµР°РЅСЃР° РІ Р‘Р”
2017-10-31 18:47:24 [DEBUG][AuthenticationService] [isUserAuthentification:49] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, Р°СѓС‚РµРЅС‚РёС„РёС†РёСЂРѕРІР°РЅ Р»Рё РїРѕР»СЊР·РѕРІР°С‚РµР»СЊ
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [isUserAuthorization:28] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, Р°РІС‚РѕСЂРёР·РѕРІР°РЅ Р»Рё РїРѕР»СЊР·РѕРІР°С‚РµР»СЊ
2017-10-31 18:47:24 [INFO ][DataContextDAO] [isUserRoleValid:106] - Р РѕР»СЊ РІРµСЂРЅР°
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [isUserAccounting:72] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, РІС‹РїРѕР»РЅРµРЅ Р»Рё РїСЂРѕС†РµСЃСЃ Р°РєРєР°СѓРЅС‚РёРЅРіР°
2017-10-31 18:47:24 [ERROR][DataContextDAO] [isDateDsAndDeValid:125] - Р”Р°С‚Р°(С‹) РЅРµ СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓРµС‚ С„РѕСЂРјР°С‚Сѓ
java.time.format.DateTimeParseException: Text 'xxxx-10-19' could not be parsed at index 0
	at java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:1949) ~[?:1.8.0_101]
	at java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1851) ~[?:1.8.0_101]
	at java.time.LocalDate.parse(LocalDate.java:400) ~[?:1.8.0_101]
	at java.time.LocalDate.parse(LocalDate.java:385) ~[?:1.8.0_101]
	at general.serv.DataValidator.isDateDsAndDeValid(DataValidator.java:122) [classes/:?]
	at general.serv.AuthorizationService.isUserAccounting(AuthorizationService.java:74) [classes/:?]
	at AccountingServiceTest.accountinTestInvalidDate(AccountingServiceTest.java:125) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42) [junit-4.4.jar:?]
	at org.junit.runner.JUnitCore.run(JUnitCore.java:130) [junit-4.4.jar:?]
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117) [junit-rt.jar:?]
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42) [junit-rt.jar:?]
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262) [junit-rt.jar:?]
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84) [junit-rt.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147) [idea_rt.jar:?]
2017-10-31 18:47:24 [DEBUG][AuthenticationService] [isUserAuthentification:49] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, Р°СѓС‚РµРЅС‚РёС„РёС†РёСЂРѕРІР°РЅ Р»Рё РїРѕР»СЊР·РѕРІР°С‚РµР»СЊ
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [isUserAuthorization:28] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, Р°РІС‚РѕСЂРёР·РѕРІР°РЅ Р»Рё РїРѕР»СЊР·РѕРІР°С‚РµР»СЊ
2017-10-31 18:47:24 [INFO ][DataContextDAO] [isUserRoleValid:106] - Р РѕР»СЊ РІРµСЂРЅР°
2017-10-31 18:47:24 [DEBUG][AuthorizationService] [isUserAccounting:72] - РџСЂРѕРІРµСЂРєР° РЅР° С‚Рѕ, РІС‹РїРѕР»РЅРµРЅ Р»Рё РїСЂРѕС†РµСЃСЃ Р°РєРєР°СѓРЅС‚РёРЅРіР°
2017-10-31 18:47:24 [INFO ][DataContextDAO] [isDateDsAndDeValid:128] - Р”Р°С‚Р°(С‹) СЃРѕРѕС‚РІРµС‚СЃС‚РІСѓСЋС‚ С„РѕСЂРјР°С‚Сѓ
2017-10-31 18:47:24 [ERROR][DataContextDAO] [isVolumeValid:143] - Р—РЅР°С‡РµРЅРёРµ РѕР±СЉС‘РјР° xxx РІРІРµРґРµРЅРѕРј РЅРµРєРѕСЂСЂРµРєС‚РЅРѕ
java.lang.NumberFormatException: For input string: "xxx"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[?:1.8.0_101]
	at java.lang.Integer.parseInt(Integer.java:580) ~[?:1.8.0_101]
	at java.lang.Integer.valueOf(Integer.java:766) ~[?:1.8.0_101]
	at general.serv.DataValidator.isVolumeValid(DataValidator.java:141) [classes/:?]
	at general.serv.AuthorizationService.isUserAccounting(AuthorizationService.java:74) [classes/:?]
	at AccountingServiceTest.accountinTestInvalidVolume(AccountingServiceTest.java:146) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42) [junit-4.4.jar:?]
	at org.junit.runner.JUnitCore.run(JUnitCore.java:130) [junit-4.4.jar:?]
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:117) [junit-rt.jar:?]
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:42) [junit-rt.jar:?]
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:262) [junit-rt.jar:?]
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:84) [junit-rt.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147) [idea_rt.jar:?]
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isUserRoleValid:106] - Роль верна
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAccounting:72] - Проверка на то, выполнен ли процесс аккаунтинга
2017-11-01 11:59:39 [ERROR][DataContextDAO] [isDateDsAndDeValid:125] - Дата(ы) не соответствует формату
java.time.format.DateTimeParseException: Text 'xxxx-10-19' could not be parsed at index 0
	at java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:1949) ~[?:1.8.0_101]
	at java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1851) ~[?:1.8.0_101]
	at java.time.LocalDate.parse(LocalDate.java:400) ~[?:1.8.0_101]
	at java.time.LocalDate.parse(LocalDate.java:385) ~[?:1.8.0_101]
	at general.serv.DataValidator.isDateDsAndDeValid(DataValidator.java:122) [classes/:?]
	at general.serv.AuthorizationService.isUserAccounting(AuthorizationService.java:74) [classes/:?]
	at AccountingServiceTest.accountinTestInvalidDate(AccountingServiceTest.java:125) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42) [junit-4.4.jar:?]
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252) [surefire-junit4-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141) [surefire-junit4-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112) [surefire-junit4-2.12.4.jar:2.12.4]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189) [surefire-api-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75) [surefire-booter-2.12.4.jar:2.12.4]
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isUserRoleValid:106] - Роль верна
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAccounting:72] - Проверка на то, выполнен ли процесс аккаунтинга
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isDateDsAndDeValid:128] - Дата(ы) соответствуют формату
2017-11-01 11:59:39 [ERROR][DataContextDAO] [isVolumeValid:143] - Значение объёма xxx введеном некорректно
java.lang.NumberFormatException: For input string: "xxx"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65) ~[?:1.8.0_101]
	at java.lang.Integer.parseInt(Integer.java:580) ~[?:1.8.0_101]
	at java.lang.Integer.valueOf(Integer.java:766) ~[?:1.8.0_101]
	at general.serv.DataValidator.isVolumeValid(DataValidator.java:141) [classes/:?]
	at general.serv.AuthorizationService.isUserAccounting(AuthorizationService.java:74) [classes/:?]
	at AccountingServiceTest.accountinTestInvalidVolume(AccountingServiceTest.java:146) [test-classes/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.junit.internal.runners.TestMethod.invoke(TestMethod.java:59) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTestMethod(MethodRoadie.java:98) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie$2.run(MethodRoadie.java:79) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:87) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:77) [junit-4.4.jar:?]
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:42) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.invokeTestMethod(JUnit4ClassRunner.java:88) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.runMethods(JUnit4ClassRunner.java:51) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner$1.run(JUnit4ClassRunner.java:44) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:27) [junit-4.4.jar:?]
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:37) [junit-4.4.jar:?]
	at org.junit.internal.runners.JUnit4ClassRunner.run(JUnit4ClassRunner.java:42) [junit-4.4.jar:?]
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252) [surefire-junit4-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141) [surefire-junit4-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112) [surefire-junit4-2.12.4.jar:2.12.4]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_101]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_101]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_101]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_101]
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189) [surefire-api-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115) [surefire-booter-2.12.4.jar:2.12.4]
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75) [surefire-booter-2.12.4.jar:2.12.4]
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isUserRoleValid:106] - Роль верна
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAccounting:72] - Проверка на то, выполнен ли процесс аккаунтинга
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isDateDsAndDeValid:128] - Дата(ы) соответствуют формату
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isVolumeValid:146] - Значение объёма введенно корректно!
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [addAccounting:52] - Добавление пользовательского сеанса в БД
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [ERROR][DataContextDAO] [hasGetUserLoginDAO:30] - Пользователь не найден по логину xxx
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [ERROR][DataContextDAO] [hasGetUserPasswordDAO:52] - Хэши паролей не совпадают!
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isUserRoleValid:106] - Роль верна
2017-11-01 11:59:39 [ERROR][DataContextDAO] [hasResUserAccessDAO:68] - Пути к ресурсу a.b не существует!
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isUserRoleValid:106] - Роль верна
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [INFO ][DataContextDAO] [isUserRoleValid:106] - Роль верна
2017-11-01 11:59:39 [ERROR][DataContextDAO] [hasResUserAccessDAO:68] - Пути к ресурсу xxx не существует!
2017-11-01 11:59:39 [DEBUG][AuthenticationService] [isUserAuthentification:49] - Проверка на то, аутентифицирован ли пользователь
2017-11-01 11:59:39 [DEBUG][AuthorizationService] [isUserAuthorization:28] - Проверка на то, авторизован ли пользователь
2017-11-01 11:59:39 [ERROR][DataContextDAO] [isUserRoleValid:110] - Роль пользователя xxx не подходит ни к одному значению из коллекции ролей
